src/main/java/duke/commands/ExitCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.commands;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Command to exit program.
+ */
+public class ExitCommand extends Command {
+    /**
+     * ExitCommand constructor.
+     */
+    public ExitCommand() {
```

Comment:
Why declare a constructor that will be the same as the default one? Just a small nitpick.